src/main/java/ladder/domain/LadderRow.java

```diff
@@ -0,0 +1,35 @@
+package ladder.domain;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class LadderRow {
+    private Boolean[] connectedPoints;
+    private static final Random random = new Random();
+
+    public LadderRow(Boolean[] connectedPoints) {
+        this.connectedPoints = connectedPoints;
+    }
+
+    public LadderRow(int size) {
+        connectedPoints = new Boolean[size];
+        connectedPoints = Arrays.stream(connectedPoints).map(point -> random.nextBoolean()).toArray(Boolean[]::new);
```

Comment:
```suggestion
        connectedPoints = Arrays.stream(connectedPoints)
        .map(point -> random.nextBoolean())
        .toArray(Boolean[]::new);
```
스트림은 점 하나의 하나씩 사용하는게 좋습니다.